WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewView.swift

```diff
@@ -99,6 +101,7 @@ struct WooPaymentsDepositsCurrencyOverviewView: View {
                 .padding(.bottom)
             }
         }
+        .safariSheet(isPresented: $showDepositSummaryInfo, url: WooConstants.URLs.wooPaymentsDepositSchedule.asURL())
```

Comment:
Adding that URL to the view model is a minor thing to consider. That way we can test it and remove logic from the view.

Reply:
Sure, I'll take a look at this if there's time later. It's one of those things where it's so simple that arguably keeping it in the view is clearer though, which was why I left it here.